app.css

```diff
@@ -22,3 +24,222 @@ a {
   text-decoration: none;
   color: inherit;
 }
+
+.section-hero {
+background: url("img/img_hero.jpeg") center center no-repeat;
+background-size: cover;
+max-width: 100%;
+min-height: 50rem;
+font-family: "Amatic SC", sans-serif;
+}
+
+.container__section-body {
+  margin: auto;
+  max-width: 60%;
+}
+
+.section-body {
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  min-height: 48rem;
+}
+
+.top-nav {
+  margin: 1rem 2rem;
+}
+
+.nav_links {
+  display: flex;
+  margin: 1rem;
+}
+
+.nav_link-items {
+  margin: 0 1rem;
+}
+
+.jumbotron-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin: auto;
+}
+
+.jumbo-text {
+  font-size: 4rem;
+  text-align: center;
+}
+
+.small-text {
+  font-size: 1rem;
+  margin: -3rem 0rem 4rem 0rem;
+}
+
+.container__nav {
+  display: flex;
+  justify-content: space-between;
+  max-width: 65rem;
+  margin: auto;
+  padding: -1rem;
+}
+
+
+.post-title {
+  font-size: 2rem;
+  text-align: center;
+}
+
+.author-photo__post {
+  border-radius: 100%;
+  max-width:  50%;
+  height: 6vh;
+  margin: 1rem;
+}
+
+.date-time {
+  text-decoration: underline;
+}
+
+.post-quote {
+  font-size: 2rem;
+}
+
+.container__post {
+  margin: 1rem;
+  width: calc(100% * 2 / 3 - 1rem - 1rem);
+  min-width: 20rem;
+  flex-grow: 1;
+}
+
+.container__author {
+  margin: 1rem;
+}
+
+.author {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.btn {
+  color: grey;
+}
+
+.btn__selected {
+  color: #a66e0e;
+}
+
+.sidebar {
+  text-align: center;
+}
+
+.container__sidebar {
+  width: calc(100% * 1 / 3 - 1rem - 1rem);
+  margin: 1rem;
+  flex-grow: 1;
+}
+
+.divider {
+  width: 73%;
+  margin: 2rem auto;
+}
+
+.author-photo {
+  margin: 1rem 0 0 0;
+  border-radius: 100%;
+}
+
+.author-name {
+  font-size: 1rem;
+  margin: 0;
+}
+
+.social__media-item {
+  margin: 0 0 0 .5rem;
+  color: #3e4147;
+}
+
+.social__media-list {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 3rem;
+  font-size: .5rem;
+}
+
+.image__sidebar {
+  font-size: .6rem;
+}
+
+.ad-link {
+  display: flex;
+  justify-content: flex-end;
+  text-decoration: underline;
+}
+
+.sidebar-content {
+  text-align: left;
+}
+
+.container__section-footer {
+  margin: -1rem;
+}
+
+.footer-title {
+  font-size: 3rem;
+  text-align: center;
+}
+
+.container__footer-img {
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  flex-wrap: wrap;
+}
+
+.footer-img {
+  width: calc(100% * 1 / 4);
+  margin: 0;
+  min-width: 21rem;
+  position: relative;
+  flex-grow: 1;
+}
+
+.footer-img img {
+  display: block;
+  width: 100%;
+}
+
+.img-header {
```

Comment:
I think one thing that is making this hard to position is the default margin.

Once you do this, use top and left if you want the text to be pinned close to the top left edge.